test_rclcpp/test/test_local_parameters.cpp

```diff
@@ -414,6 +414,41 @@ TEST(CLASSNAME(test_local_parameters, RMW_IMPLEMENTATION), get_parameter_or) {
   }
 }
 
+TEST(CLASSNAME(test_local_parameters, RMW_IMPLEMENTATION), get_parameter_or_set_default) {
+  using rclcpp::Parameter;
+
+  auto node = rclcpp::Node::make_shared("test_parameters_get_parameter_or_set_default");
+  auto set_parameters_results = node->set_parameters({
+    Parameter("foo", 2),
+  });
+
+  // Check to see if they were set.
+  for (auto & result : set_parameters_results) {
+    ASSERT_TRUE(result.successful);
+  }
+
+  {
+    // try to get with default a parameter that is already set
+    int64_t foo_value = -1;
+    node->get_parameter_or("foo", foo_value, static_cast<int64_t>(42));
```

Comment:
Is this supposed to be a call to `get_parameter_or_set_default()`?

Reply:
Yes, absolutely, thanks for pointing it out.  Cut-n-paste error.  It should be fixed now.